### PR TITLE
feat: up related depend for orbit23

### PR DESCRIPTION
**Pull request by zachferland**

### CODE REVIEW

**Comment by oed** [comment]
This also requires https://github.com/orbitdb/ipfs-log/pull/292 I presume

**Comment by zachferland** [comment]
@oed all set, an added release commit here

**Review by oed — APPROVED** [review]
This should be fine to release before it's updated in 3box-js right?

**Comment by oed** [comment]
If not, we should bump the minor version number.

**Comment by zachferland** [comment]
yeah, but would be nice to minimize, maybe will just wait till closer to release 
minor would unfortunately get pulled in as well based on the definition in 3boxjs

**Comment by oed** [comment]
Ok, then lets try to synchronize the releases :)

**Comment by oed** [comment]
Sorry I totally forgot that we needed to merge and release this before releaseing 3box-js. Should not cause any issues right now though.

**Comment by zachferland** [comment]
Ah yeah! no should be good

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Release Notes
 
+## v1.0.7 - 2020-1-14
```
Comment (oed) [inline RELEASE-NOTES.md:3]: Typo, Can you fix this in #12 @zachferland ?

**Review by oed — APPROVED** [review]

**Pull request merged**